build/yaml/botbuilder-dotnet-ci.yml

```diff
@@ -6,7 +6,7 @@
 name: $(Build.BuildId)
 
 pool:
-  vmImage: $[ coalesce( variables['VMImage'], 'windows-2019' ) ] # or 'windows-latest' or 'vs2017-win2016'
+  vmImage: $[ coalesce( variables['VMImage'], 'windows-2022' ) ] # or 'windows-latest' or 'vs2017-win2016'
   demands:
```

Comment:
vmImage 'vs2017-win2016' is no longer supported. This should be removed.

Reply:
Comment updated. Thank!